tools/cleanup-cinder-attachments.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -u
-for vol in $(openstack volume list| grep juju| grep in-use| awk '{print $2}'); do
+
```

Comment:
If we are going to ignore name we should do ```openstack volume list --status in-use```

Reply:
Yes, that is simpler.